src/service/QueryInstance.cpp

```diff
@@ -142,6 +142,7 @@ void QueryInstance::onError(Status status) {
         case Status::Code::kIndexNotFound:
         case Status::Code::kInserted:
         case Status::Code::kKeyNotFound:
+        case Status::Code::kPartialSuccess:
```

Comment:
When return this error?

Reply:
Currently this error will be processed as ErrorCode::E_EXECUTION_ERROR in QueryInstance